airflow/www/templates/airflow/graph.html

```diff
@@ -79,24 +79,55 @@
     {{ super() }}
 
     <script src="{{ url_for('static', filename='d3.v3.min.js') }}"></script>
-    <script src="{{ url_for('static', filename='dagre-d3.js') }}"></script>
+    <script src="http://cpettitt.github.io/project/dagre-d3/latest/dagre-d3.js"></script>
```

Comment:
In closed environments where Airflow cannot go outside this will raise an error.